src/components/html-injection/ko/runtime/widget-runtime.html

```diff
@@ -0,0 +1,8 @@
+<iframe
+    data-bind="attr: { 
+        srcdoc: htmlCode,
+        style: `height: ${htmlCodeSizeStyles.height}; width: ${htmlCodeSizeStyles.width}`,
```

Comment:
@azaslonov you've mentioned some better way how this could be done

Reply:
Yep, there is a `styled` binding handler. You can see its usage example here: https://github.com/paperbits/paperbits-core/blob/master/src/button/ko/button.html, and filling the styles for it here: https://github.com/paperbits/paperbits-core/blob/master/src/button/ko/buttonViewModelBinder.ts#L36